### PR TITLE
Create User.cs

### PR DESCRIPTION
**Pull request by Davilagp**

### CODE REVIEW

File: CSD412webProject/CSD412webProject/Models/User.cs [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace CSD412webProject.Models
+{
+    public class User
+    {
+
+        public int Id { get; set; }
```
Comment (ksakovich) [inline CSD412webProject/CSD412webProject/Models/User.cs:12]: Not sure that we need a setter for an ID ( I would prefer to assign it while initialization (creation of the object), than leaving it blank at the beginning , and then set it (someday in future)

File: CSD412webProject/CSD412webProject/Models/User.cs [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+        public string Name { get; set; }
+        public string LastName { get; set; }
+        public string Email { get; set; }
+        public DateTime DateOfBirth => DateOfBirth.Date;
+        public string Role { get; set; }
+
+        public User()
+        {
```
Comment (ksakovich) [inline CSD412webProject/CSD412webProject/Models/User.cs:20]: I would put here also ID that getting auto incremental static int IdCounter (this.Id = IdCounter++;

File: CSD412webProject/CSD412webProject/Models/User.cs [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace CSD412webProject.Models
+{
+    public class User
+    {
+
```
Comment (ksakovich) [inline CSD412webProject/CSD412webProject/Models/User.cs:11]: could you please ad private static int IdCounter as a filed = 0

File: CSD412webProject/CSD412webProject/Models/User.cs [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+        public int Id { get; set; }
+        public string Name { get; set; }
+        public string LastName { get; set; }
+        public string Email { get; set; }
+        public DateTime DateOfBirth => DateOfBirth.Date;
+        public string Role { get; set; }
+
```
Comment (ksakovich) [inline CSD412webProject/CSD412webProject/Models/User.cs:18]: missing list watch later
missing list my favorite movies

File: CSD412webProject/CSD412webProject/Models/User.cs [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+        public User()
+        {
+            this.Name = "Default";
+            this.LastName = "Default";
+            this.Email = "Default";
```
Comment (ksakovich) [inline CSD412webProject/CSD412webProject/Models/User.cs:23]: please change it for "default@email.com" (or something like that) - we might have in the future have a checker for '@' and '.' characters in the email (regexp)

**Review by ksakovich — CHANGES_REQUESTED** [review]
left some comments what need to be fixed
Thanks

File: CSD412webProject/CSD412webProject/Models/User.cs (hunk 4 above)
Comment (Davilagp) [reply CSD412webProject/CSD412webProject/Models/User.cs:18]: We don't have "list" model yet.

Comment (ksakovich) [reply CSD412webProject/CSD412webProject/Models/User.cs:22]: that's a good point

File: CSD412webProject/CSD412webProject/Models/User.cs [hunk 6]
```diff
@@ -1,27 +1,35 @@
...
+            this.Id = IdCounter++;
             this.Name = "Default";
             this.LastName = "Default";
-            this.Email = "Default";
-            this.Role = "Default";
+            this.Email = "default@email.com";
+
         }
+
+
```
Comment (ksakovich) [inline CSD412webProject/CSD412webProject/Models/User.cs:32]: nit: get rid of these 3 empty lines and push it

File: CSD412webProject/CSD412webProject/Models/User.cs [hunk 7]
```diff
@@ -1,27 +1,35 @@
...
 
         public User()
         {
+            this.Id = IdCounter++;
             this.Name = "Default";
             this.LastName = "Default";
-            this.Email = "Default";
-            this.Role = "Default";
+            this.Email = "default@email.com";
+
```
Comment (ksakovich) [inline CSD412webProject/CSD412webProject/Models/User.cs:29]: could you please still add Role = User/Customer/defaultUser (not admin) by default

**Review by ksakovich — APPROVED** [review]

**Pull request merged**